environ/utils/data_loader.py

```diff
@@ -37,3 +37,25 @@ def load_data(
     df_merged.rename(columns=rename_dict, inplace=True)
 
     return panel_main.merge(df_merged[data_col], **kwargs)
+
+
+def load_data_herf(
```

Comment:
This looks almost exactly like the merge main panel function - is it necessary to create another function? If you create one function for one specific case, it completely defeat the purpose of a function (which is to be called and reused)

Reply:
Make sense.